### PR TITLE
Clarify IETF liaison

### LINKED ISSUE #52: clarify IETF liaison
> Internet Engineering Task Force Security Area Directorate - To coordinate broad horizontal reviews on the output of the Working Group among the security working groups at IETF.

That's not what IETF's SecDir does.  If you can clarify a bit more about what you're wanting from the IETF, I might be able to steer you.  

Who, specifically, does the WG envision doing IETF outreach?  (This does not need to be in the charter, but it could again help clarify what outreach you're intending.)

[I wonder if we should we asking more about who's going to do each of the liaison things listed in a charter, if only as a way to validate that we really intend to do them.  But that's not an issue for today.]

### PR DESCRIPTION
**Pull request by brentzundel**
fixes #52 
Signed-off-by: Brent Zundel <brent.zundel@gmail.com>

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-wg-charter/pull/80.html" title="Last updated on Mar 1, 2022, 9:29 PM UTC (3273ea9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-wg-charter/80/6ad9969...brentzundel:3273ea9.html" title="Last updated on Mar 1, 2022, 9:29 PM UTC (3273ea9)">Diff</a>

### CODE REVIEW

**Review by kdenhartog — APPROVED** [review]

File: index.html
Comment (msporny) [inline index.html]: The removed \<dd> is problematic. The SAAG won't do broad horizontal reviews, will it @samuelweiler? The reason the SAD was in there before was to tell us "Oh, you want X... then you need to talk with Y." Will the SAAG do that for us now (I'm not familiar w/ them)?

I guess the question is -- who is going to tell us "If you want that sort of review, you need to go /over there/."?

**Review by selfissued — CHANGES_REQUESTED** [review]
To my knowledge, neither the IESG nor SAAG have formal liaison relationships, so it doesn't make sense to include them here.  Please remove them.

That said, when we think we're ready, I'm great with informally asking either or both if they'd be willing to do a friendly review, possibly assisted by Mark Nottingham, who had expressed interest in helping in the past.

**Comment by msporny** [comment]
@mnot, what would be the best way to say "We would like to work with the 'Security People' at IETF when it comes to reviewing, or suggesting who should review, the normative output for this W3C Working Group?".

This is what's in the charter right now under the Liasons section (I added the liaisons after speaking with you months ago about what to do):

[Internet Engineering Task Force Security Area Directorate](https://datatracker.ietf.org/group/secdir/about/)
To coordinate broad horizontal reviews on the output of the Working Group among the security working groups at IETF.
[Internet Engineering Task Force Crypto Forum Research Group](https://datatracker.ietf.org/rg/cfrg/about/)
To perform broad horizontal reviews on the output of the Working Group and to ensure that new pairing-based and post-quantum cryptographic algorithms and parameters can be integrated into the Data Integrity ecosystem.

@samuelweiler wanted us to replace the SAD with the SAAG. @selfissued is saying don't list the liaison relationship at all. 

Help, please.

**Comment by mnot** [comment]
Perhaps it's best to say something a bit more fuzzy, like:

Internet Engineering Task Force
The Working Group will seek security review from the IETF, coordinated through the Liaison

**Comment by brentzundel** [comment]
@selfissued @msporny @samuelweiler 
I have revised the PR according to the suggestions from @mnot, please review

**Review by selfissued — APPROVED** [review]
Fine now - thanks

**Review by msporny — APPROVED** [review]
I'm fine w/ being less precise.

**Review by Sakurann — APPROVED** [review]

**Comment by iherman** [comment]
@samuelweiler, this PR (and the linked issue) is an answer to your comment; are you o.k. with this, so that we can merge this PR and close the corresponding issue?

**Comment by brentzundel** [comment]
Merging this PR in accordance with our current working mode. If further changes are desired, we can track those in the related issue #52

**Pull request merged**
